content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+== WebAuthn primer for mobile development
+This section should be referenced if you are looking to begin development for a mobile application, but are unfamiliar with WebAuthn concepts and development. The goal of this page is threefold:
+
+* Point you in the direction of  resources to learn more about the WebAuthn protocol
+* Provide additional considerations for developing WebAuthn on mobile devices
+* Guide developers on how to deploy an example WebAuthn application.
+
+=== Where can I learn more about WebAuthn?
```

Comment:
Typically, the learn more section is at the bottom of the content. Consider moving to the bottom of the page